app/license.go

```diff
@@ -379,7 +379,10 @@ func (s *Server) GenerateRenewalToken(expiration time.Duration) (string, *model.
 
 	currentToken, _ := s.Store.System().GetByName(model.SystemLicenseRenewalToken)
 	if currentToken != nil {
-		tokenIsValid, _ := s.renewalTokenValid(currentToken.Value, license.Customer.Email)
+		tokenIsValid, err := s.renewalTokenValid(currentToken.Value, license.Customer.Email)
+		if err != nil {
+			mlog.Warn("error checking license renewal token validation", mlog.Err(err))
+		}
```

Comment:
I guess the question here comes down to:
1. Why do we store the token in the System Store? It seems that we want to generate a new token for a link on demand and here if there is an error validating - we ignore and generate a new one anyways.

Reply:
I don't mind either ways. Technically, it seems fine to return an existing token if it hasn't expired. I guess the main question we had was "Why are we getting a spam of these token renewal requests during load-tests"? Do we know why is that?

Reply:
+1, this should only be called if we are sending the license expired email or in the front end calling the endpoint to get renewal link. Are any of these actions happening in load testing?

Reply:
@agnivade the spam of tokens was due an attempt to generate a token for each admin when sending the email. So if multiple admins exist then we get several of those logs.

@marianunez my thought also was we don’t need to store the token in the db. We can always have one generated on the fly

Reply:
>  the spam of tokens was due an attempt to generate a token for each admin when sending the email. So if multiple admins exist then we get several of those logs.

Why is the load test triggering these emails?

Reply:
I think it was getting triggered by `ExpiryNotify` job.

Reply:
@isacikgoz not sure I understand how it does. We trigger this job as a recurring task

Reply:
We need to get to the bottom of this as this is blocking the release. @isacikgoz - The question I had is do we see the logs being repeated or is it a one time spam of logs and it stops? I am wondering if we can create a separate Terraform instance and hand it off to @AGMETEOR to debug it further?

I guess you could not reproduce this locally right?

Reply:
Yes, I couldn't reproduce it locally.

Reply:
@AGMETEOR actually you can first try the load test agent on your local instance, maybe we wouldn't need to go over the aws path? WDYT?

Reply:
Oh, can we reproduce this with a local load test? Then it should be fairly easy to debug.

Reply:
@AGMETEOR / @isacikgoz -  Do we want to just move on with the review of the PR given that we weren't able to reproduce this a second time?

Reply:
Yes.